app/models/fae/navigation.rb

```diff
@@ -58,7 +58,7 @@ def increment_coordinates
     end
 
     def current_section
-      @current_path.gsub(/\/new|\/[0-9]\/edit/, '')
+      @current_path.gsub(/\/new|\/[0-9]*\/edit/, '')
```

Comment:
Do you mean `*` or `+` ? `*` is going to include characters not numeric. Alternately, `\d+`